### PR TITLE
fix: KAN-242 pnpm cache

### PR DESCRIPTION
**Pull request by andriiholubenko**

### CODE REVIEW

**Comment by ryan-astrumu** [comment]
Did we not move entirely away from yarn? Where is it still used?

File: dependencies/action.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    - name: yarn dependencies
+      shell: bash
+      if: ${{ inputs.packageManager == 'yarn' && steps.yarn_cache.outputs.cache-hit != 'true' }}
+      run: ${{ inputs.packageManager }} install --frozen-lockfile --check-files --ignore-scripts
+
+    - name: pnpm dependencies
+      shell: bash
+      if: inputs.packageManager == 'pnpm'
+      run: ${{ inputs.packageManager }} install --frozen-lockfile
```
[the remark this answers is not in the document]
Comment (ryan-astrumu) [reply dependencies/action.yml:34]: It is likely a separate set of work to fix it, though.

[the remark this answers is not in the document]
Comment (vtereshyn) [reply dependencies/action.yml:34]: it should fail if the lockfile is out of date. each developer should run `pnpm i` locally and push changed lockfile. as you can see build fails only in dependabot PRs that are created automatically

**Comment by vtereshyn** [comment]
@ryan-astrumu no, we didn't. we use `yarn` in `apps` repository

**Review by yevhenii-r — APPROVED** [review]

**Pull request merged**